python/grass/pygrass/modules/interface/module.py

```diff
@@ -566,6 +566,16 @@ def __init__(self, cmd, *args, **kargs):
             if e.tag not in {"parameter", "flag"}:
                 self.__setattr__(e.tag, GETFROMTAG[e.tag](e))
 
+        # some modules may have a label defined: in this case stay
+        # consistent with other modules -> define 'description'
+        # property instead
+        if hasattr(self, "description") is False:
+            if hasattr(self, "label") is True:
```

Comment:
Isn't `if hasattr(...)` and `if not hasattr(...)` more readable here than `if hasattr(...) is True/False`?